database/migrations/2021_04_21_102707_change_statuses_to_accept_business_checkin_as_integer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeStatusesToAcceptBusinessCheckinAsInteger extends Migration
+{
+    public function up() {
+        Schema::table('statuses', function(Blueprint $table) {
+            $table->integer('business')
+                  ->default(0)
+                  ->change();
```

Comment:
```suggestion
            $table->unsignedTinyInteger('business')
                  ->default(0)
                  ->change();
```
Storage space costs money! xD

An unsigned tiny integer is enough. The range goes from 0 to 255.